src/libs/getCurrentPosition/index.ts

```diff
@@ -0,0 +1,19 @@
+import {GeolocationErrorCode, GetCurrentPosition} from './getCurrentPosition.types';
+
+const getCurrentPosition: GetCurrentPosition = (success, error, options) => {
+    if (typeof navigator === 'undefined' || !('geolocation' in navigator)) {
```

Comment:
```suggestion
    if (_.isUndefined(navigator) || !lodashHas(navigator, 'geolocation')) {
```
This is more consistent with the rest of the codebase.

Reply:
why can't we use `navigator === undefined`?

@tgolen I think the new approach we're taking is to use fewer lodash or underscore methods if there is a native JavaScript way of doing the same thing
. I think it's part of TS guideline so strictly speaking, it's not applicable here, but it makes sense to use native approach when native approach is already succinct

Reply:
@huzaifa-99 Hayata and Rory brought me quickly up to speed here and I wasn't aware yet that we were going away from underscore and lodash. Please revert the changes here back to what they were originally without using underscore. Sorry about that!